INSTALL.md

```diff
@@ -31,6 +31,12 @@ The sympton will be recognizable by errors during an `npm install` that contains
 
 * Install MongoDB (Version 2.4 or greater) [http://www.mongodb.org/downloads](http://www.mongodb.org/downloads)
 
+### ImageMagick
+
+The ImageMagick command line tools are used to convert images. If you're seeing
+`Uncaught Error: spawn convert ENOENT` when running the integration tests, make
```

Comment:
Das kannst Du so allgemein nicht sagen, den Node-Fehler gibt's immer, wenn eine externe Applikation fehlt. Du solltest noch erwähnen, in welchen Tests konkret.